app/src/pages/inside/common/launchSuiteGrid/defectStatistics/defectStatistics.jsx

```diff
@@ -34,6 +34,10 @@ export class DefectStatistics extends Component {
     projectDefects: PropTypes.object,
     data: PropTypes.object,
     customProps: PropTypes.object,
+    launchesLimit: PropTypes.number,
+    compositeAttribute: PropTypes.string,
+    isLatest: PropTypes.bool,
+    filterType: PropTypes.bool,
```

Comment:
It would be nice to hold all such parameters in one object and use `spread` where it should be destructured.